src/store/actions/api.actions.js

```diff
@@ -240,18 +152,42 @@ export const fetchEndpoint = pathname => async dispatch => {
                     return plugin;
             }
         });
+
         const lens = R.lensPath(['plugins']);
-        const preparedApi = R.set(lens, preparedPlugins, response.data);
+        const cloning = obj => flag => R.dissoc('name', obj);
+        const notCloning = obj => flag => flag === false && obj;
+        const adjustIfCloning = obj => {
+            const makeDecision = R.either(
```

Comment:
WDYT of return first approach:
```js
return R.either(
  notCloning(obj),
  cloning(obj)
)(!!flag)
```

or

```js
const adjustIfCloning = obj =>
  R.either(
    notCloning(obj),
    cloning(obj)
  )(!!flag)
```

Reply:
You are the best what happened to me, caraca 🤣  !!! @rafaeljesus